controllers/customers.js

```diff
@@ -0,0 +1,48 @@
+const models = require('../models')
+const Customer = require('../models/customer')
+
+
+const handleUploadCustomers = customers => {
+  const customerEmailValidate = customers.map(customer => {
+    const { email } = customer
+    // removes whitespaces and commas
+    const invalidChars = /[,\s]+|[,\s]+/g
```

Comment:
This regular expression only checks for commas and white spaces.
What about the following regular expression? 
```
/[,]+|[.]{2,}|\s
```